src/main/resources/static/templates/Http3xxStatusCodeBasedInjection/LEVEL_1/Http3xxStatusCodeBasedInjection.js

```diff
@@ -1,7 +1,13 @@
 function updatePlaceholderDiv() {
   let placeholderAnchorElement = document.getElementById("placeholder");
-  placeholderAnchorElement.href =
-    getUrlForVulnerabilityLevel() + "?returnTo=/VulnerableApp";
+  let url = window.location.href;
+  if (url.endsWith(":9090/VulnerableApp/")){
```

Comment:
I think it is better to just check for `/VulnerableApp/`. Modifying port is easier and should not be hardcoded.